### PR TITLE
Resolve Breaking Aggregate Reports

### PR DESCRIPTION
**Pull request by scbedd**
This is because `analyze` is being utilized. Just need to set variables to skip the `mypy` and `pylint` steps.

@mitchdenny

### CODE REVIEW

File: eng/pipelines/templates/jobs/aggregate-reports.yml [hunk 1]
```diff
@@ -11,6 +11,10 @@ resources:
 trigger: none
```
Comment (weshaggard) [inline eng/pipelines/templates/jobs/aggregate-reports.yml:11]: Orthogonal but generally pipeline entry-points shouldn't be under the templates folder as they aren't a reusable template. I've mostly been putting pipelines at in `eng/pipelines` folder directly.

Comment (scbedd) [reply eng/pipelines/templates/jobs/aggregate-reports.yml:11]: Yeah this is a good suggestion. I'll move it with this PR. There are additional changes I need to make as the conditional injection I'm using simply doesn't work if it's not isolated to a single job. :/

Comment (scbedd) [reply eng/pipelines/templates/jobs/aggregate-reports.yml:11]: @weshaggard I tested the job prior to the commit moving it. [Works as intended](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=414186&view=logs&j=0874ffeb-2919-5b4b-20de-16a97970b94c)

MY adding of commits cleared your approval. When you see this comment can you approve this again?

Comment (scbedd) [reply eng/pipelines/templates/jobs/aggregate-reports.yml:11]: I'll update the actual build def to point at the new location after I merge it to master.

**Pull request merged**